executor/benchmark_test.go

```diff
@@ -225,6 +225,7 @@ func buildMockDataSourceWithIndex(opt mockDataSourceParameters, index []int) *mo
 
 type aggTestCase struct {
 	// The test table's schema is fixed (aggCol Double, groupBy LongLong).
+
```

Comment:
This change is not necessary.

Reply:
`goword` assumed line 227 was a comment for `windowFunc`, and not start with word `windowFunc`. So I added a blank line.

Reply:
Hi, I think it's better to move this comment above `type aggTestCase`

```go
// The test table's schema is fixed (aggCol Double, groupBy LongLong).
type aggTestCase struct {
}
```